README.md

```diff
@@ -1,5 +1,30 @@
 BreathingTasks_PsychoPy
 ==============
-PsychoPy code to display instructions for a 'Breath Hold' and a 'Cued Deep Breathing' task. 
---------------
+PsychoPy code to display visual instructions for a 'Breath Hold' and a 'Cued Deep Breathing' task. 
+
 Written with PsychoPy3.0.5
+
+Tips for using this code:
+--------------
+
+- It is designed to be used alongside recordings of end-tidal CO2 and O2 via a nasal cannula, during MRI, but can be used with other set-ups. 
+- Press '5' when it says 'waiting for scanner ...' to manually start the breathing task instructions. Or set this input to be whatever the MRI sends to indicate the first volume of data is being acquired so that your breathing task will be synchronized to the start of your scan. 
+- Always practice with the participant beforehand to make sure they understand the task instructions and can achieve the desired end-tidal changes. 
```

Comment:
```suggestion
- Always practice with the participant beforehand to make sure they understand the task instructions and can achieve the desired end-tidal changes. If possible, practice while monitoring the physiological signals already, to be sure of the outcome. Pay particular attention to the **expirations preceding _and_ following** the BH: if they are not performed well, you won't be able to use the collected data.
```

Reply:
I don't know if there is something to pay *particular attention to* during the practice for the CBD!